plotting_rbfes_with_cinnabar/PlottingFreeEnergiesUsingCinnabar.ipynb

```diff
@@ -22,7 +22,7 @@
     "from pprint import pprint\n",
     "import cinnabar\n",
     "from cinnabar import plotting as cinnabar_plotting\n",
-    "from cinnabar.wrangle import FEMap"
+    "from cinnabar import femap, stats"
```

Comment:
I tried running this locally with a fresh install of cinnabar:

```
# packages in environment at /home/ialibay/software/mambaforge/install/envs/openfe:
#
# Name                    Version                   Build  Channel
cinnabar                  0.3.0              pyhd8ed1ab_0    conda-forge
```

and I'm getting an import error on `femap`